server/routes/reviewer.ts

```diff
@@ -54,8 +68,9 @@ export = function CreateReviewRoutes({ offenderService, reportDetailBuilder, rev
       )
 
       const statements = await reviewService.getStatements(await systemToken(res.locals.user.username), reportId)
+      const tab = report.status === 'COMPLETE' ? '/completed-incidents' : '/not-completed-incidents'
```

Comment:
Should use the ReportStatus constant as descibed in the other comment.

Reply:
done